packages/client/src/chat.ts

```diff
@@ -12,6 +12,12 @@ import { parseIntSafe } from "./misc";
 import * as modals from "./modals";
 import ChatMessage from "./types/ChatMessage";
 
+export enum SelfChatMessageType {
+  normal,
```

Comment:
in typescript, the values of an enum should be capitalized, see: https://www.typescriptlang.org/docs/handbook/enums.html